src/utilities/directives/scroll-tabindex/scroll-tabindex.stories.js

```diff
@@ -1,4 +1,4 @@
-import { scrollTabindex } from '@/utilities/directives/index';
+import { scrollTabindex } from '@/utilities/directives/index.ts';
```

Comment:
I believe the whole point of the `index` files is not to enter its' name when importing from the directory and to be still able to import from there. :D

Reply:
Yes, you are right. I just had an eslint error about the unresolved path. For now, I'll use a relative path without the `@` alias because that solves the problem. 🙂